src/test/elements/successfactors/user-accounts.js

```diff
@@ -9,7 +9,7 @@ const userAccountsCreatePayload = tools.requirePayload(`${__dirname}/assets/user
 const userAccountsUpdatePayload = tools.requirePayload(`${__dirname}/assets/user-accounts-update.json`);
 
 suite.forElement('humancapital', 'user-accounts', null, (test) => {
-  let id;
+  let id = '03b7ea14-2528-4138-891b-d19a0b62596a';
```

Comment:
Can we move this to the sauce file, i.e. 
```suggestion
  let id = require('core/props').getForKey('successfactors', 'userAccountId');
```